src/stories/buttons/buttons.js

```diff
@@ -1,28 +1,132 @@
 import React from "react";
 
-import { Primary, Secondary, Success, Dngr } from "./buttons.style";
+import {
+  Primary,
+  Secondary,
+  Success,
+  Dngr,
+  PrimaryOutl,
+  SecondaryOutl,
+  SuccessOutl,
+  DngrOutl,
+  Icnoutline,
+  Icnfullline,
```

Comment:
Styled component name should start from `Styled` prefix.
For example: `StyledPrimaryButton`